cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/web/OAuth20AccessTokenController.java

```diff
@@ -37,137 +28,72 @@
 public final class OAuth20AccessTokenController extends BaseOAuthWrapperController {
 
     @Autowired
-    @Qualifier("accessTokenJwtGenerator")
-    private JwtGenerator accessTokenJwtGenerator;
+    @Qualifier("defaultAccessTokenFactory")
+    private AccessTokenFactory accessTokenFactory;
 
-    /**
-     * Instantiates a new o auth20 access token controller.
-     */
-    public OAuth20AccessTokenController() {}
-
-    /**
-     * Ensure the encryption secret has been set.
-     */
-    @PostConstruct
-    public void postConstruct() {
-        CommonHelper.assertNotNull("encryptionSecret", accessTokenJwtGenerator.getEncryptionSecret());
-    }
+    @NotNull
+    @Value("${tgt.timeToKillInSeconds:7200}")
```

Comment:
do we want to keep the `AccessToken` and `TicketGrantingTicket` timeout values the same? or should we expose them as separate properties?